biome.jsonc

```diff
@@ -52,6 +52,10 @@
     "enabled": true,
     "rules": {
       "recommended": true,
+      "suspicious": {
+        // https://developer.mozilla.org/en-US/docs/Web/API/CookieStore Reevaluate in June '26 after 1y of availability.
+        "noDocumentCookie":"off"
+      },
```

Comment:
Biome didn't like that I tried to write cookies using document.cookie.
I didn't like its suggestion because the Cookie Store API is still a bit too hot to handle (Baseline circa June 2025; we need 12 months per our browser support policy).
Hence this.

Reply:
Maybe add a TODO + Jira to improve our chances of actually re-evaluating this 😁